src/sinks/util/adaptive_concurrency/tests.rs

```diff
@@ -103,7 +103,7 @@ struct TestParams {
     requests: usize,
 
     // The time interval between requests.
-    interval: Option<f64>,
```

Comment:
I don't think you want to modify this. This is unrelated to the generator.

Reply:
No, this is actually passed into a `GeneratorConfig` down below: https://github.com/vectordotdev/vector/blob/master/src/sinks/util/adaptive_concurrency/tests.rs#L405-L406

Reply:
Aha! I stand corrected. You'll need to update the test fixtures in `tests/data/adaptive-concurrency/`. It is currently causing a failure since this change makes the `interval` field required:

https://github.com/vectordotdev/vector/pull/9390/checks?check_run_id=3759479954#step:9:141

I think you'll want to specify an interval of 0 to match the previous behavior when `interval` wasn't in the fixture file.

Alternatively, you could add a serde default of 0 for the `interval` here for the concurrency tests.

Reply:
Good call. I went with a serde default here, as that seems to be the least disruptive.